src/components/navbarLinks/checkIfHeadingShouldBeRemoved.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Sjekker om headingen skal være med eller ikke
+ * @param headingValue
+ * @returns true hvis den skal fjernes. false hvis den skal inkluderes
+ */
+
+const checkIfHeadingShouldBeRemoved = (toc: TocItem) => {
+  if(toc.attributes.className === "toc-ignore") {
```

Comment:
Er `attributes` her egen greie fra TOC-generatoren? Kan kanskje være noe som dette for å støtte flere tilfeller hvor det ikke BARE er toc-ignore som klasse:

```suggestion
  if(toc.attributes.className?.includes("toc-ignore")) {
```

Reply:
Her kan det og egentlig forenkles til å returnere fremfor if / else med true false. Altså:

```ts
const checkIfHeadingShouldBeRemoved = (toc: TocItem) => {
  return toc.attributes.className?.includes("toc-ignore") ?? false;
}
```

Reply:
Blir generert fra TOC-generatoren ja